tenable/io/v3/__init__.py

```diff
@@ -1,5 +1,5 @@
 '''
-Version3API
+Version3 API
 ===========
```

Comment:
the header divider should always equal the line above it.  This follows Restructured Text guidelines.

Add a `=`